app/models/user.rb

```diff
@@ -4,8 +4,8 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
-  validates :email, presence: true
-  validates :password, presence: true
+  validates :email, uniqueness: true
```

Comment:
You should use both validations. I mean uniqueness and presence. Also, you can add validation about email's pattern.